src/targets/gpu/kernels/include/migraphx/kernels/reduce.hpp

```diff
@@ -31,7 +31,11 @@
 
 namespace migraphx {
 
-#if MIGRAPHX_HAS_DPP
+#ifdef MIGRAPHX_USE_HIPRTC
+#define MIGRAPHX_NO_DPP
+#endif
+
+#ifndef MIGRAPHX_NO_DPP
```

Comment:
Should be `#if MIGRAPHX_HAS_DPP`. ANd the above can be removed as its already handled by dpp.hpp.

Reply:
Done.